src/components/screens/monitor/delegates/delegates.js

```diff
@@ -10,22 +10,35 @@ import DelegatesTable from '../../../shared/delegatesTable';
 import MonitorHeader from '../header';
 import Tooltip from '../../../toolbox/tooltip/tooltip';
 import routes from '../../../../constants/routes';
+import Overview from './overview';
 import styles from './delegates.css';
 
 const Delegates = ({
-  delegates, t, filters, applyFilters, changeSort, sort, isMediumViewPort,
+  applyFilters,
+  changeSort,
+  chartActiveAndStandbyData,
+  chartRegisteredDelegatesData,
+  delegates,
+  filters,
+  isMediumViewPort,
+  sort,
+  standByDelegates,
+  t,
 }) => {
-  const getForgingTitle = status => ({
+  const [activeTab, setActiveTab] = useState('active');
+  const statuses = {
     forgedThisRound: t('Forging'),
     forgedLastRound: t('Awaiting slot'),
     notForging: t('Not forging'),
     missedLastRound: t('Missed block'),
-  }[status] || t('Loading'));
+  };
+
+  const getForgingTitle = status => statuses[status] || t('Loading');
```

Comment:
Why do we need to define this inside? Why does it cover some cases and not all? Why does it return an object from which we later choose a memeber? I think a simple switch-case statement covers this all and returns the value needed directly.

Reply:
we can keep this based on what we discuss